PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,6 +18,7 @@
 
 - [ ] PR title follows [Commit Convention](https://www.notion.so/godialogue/Commit-Convention-84fd9a4c149e48c998d760f1c9176df0) <!-- `feat(lang): add German language` -->
 - [ ] PR title ends with a JIRA issue ID  <!-- `fix: signup error [DIA-000]` -->
+- [ ] All relevant labels are added <!-- Labels help with PR classification -->
```

Comment:
Is it something specific to BM?

Reply:
No, it seems like other projects also use labels on their PRs (e.g. [PWA](https://github.com/dialoguemd/patient-webapp/pulls))